ingest-app/src/cmr/ingest/site/routes.clj

```diff
@@ -0,0 +1,47 @@
+(ns cmr.ingest.site.routes
+  "Defines the HTTP URL routes for the ingest web site."
+  (:require
+   [cmr.common-app.static :as static]
+   [cmr.common-app.site.data :as common-data]
+   [cmr.common-app.site.pages :as common-pages]
+   [cmr.ingest.site.pages :as pages]
+   [cmr.transmit.config :as config]
+   [compojure.core :refer [GET context routes]]
+   [ring.util.response :refer [redirect]]))
+
+(defn build-routes [system]
+  (routes
+    (context (get-in system [:ingest-public-conf :relative-root-url]) []
+      (GET "/"
+           {ctx :request-context}
+           (pages/home ctx))
```

Comment:
I am thinking a "lein generate-docs" or something similar should just generate all the pages that we can generate statically, i.e. anything except collections landing page. Is that right?